README.md

```diff
@@ -1,54 +1,64 @@
+
+## :arrow_forward: Implementation
+<p align=justify>This "website" is a crude lobotomization of what the basic DataLad datasets navigator <code>datalad create-sibling --with-ui</code> creates for you; <a href="https://datasets.datalad.org">https://datasets.datalad.org</a> is one of such websites.</p>
+<p align=justify>Also, if you check the git history, you will see that we have used <a href="http://handbook.datalad.org/en/latest/basics/101-109-rerun.html">datalad run</a> to annotate how <code>posters.json</code> was produced. So you can use <code>datalad rerun _produce_posters.json</code> (where <code>_produce_posters.json</code> is just a git tag) to regenerate <code>posters.json</code> using either a modified script or the original <code>.tsv</code>.</p> 
+
+## :computer: How to Guideline
+### :heavy_check_mark: How to Visit the OHBM2020 Posters Jitsi Rooms? 
+Go to our website: <a href="https://datalad-datasets.github.io/ohbm2020-posters/">https://datalad-datasets.github.io/ohbm2020-posters/</a> & wait for the page to load. <br><br>
+<img src="./img/datalad_OHBM2020Posters_icons.png" width="700"/>
+<br><br>
+:mag::eyes::door: 
+1. <b>Searching for a Poster</b>: Enter your desired keywords (Number ID, Title, Presenter, Category/ies) in the `Search` box, click on `Enter`, and the table will be limited to the results of your search. 
+2. <b>Viewing a Poster PDF</b>: Click on the `PDF` associated with the poster you would like to "present" or "attend". Each specific poster PDF would open in a new "dedicated" tab or window. Clicking on it again would just lead you to that tab/window if it was already open.
+3. <b>Visiting a Poster Jitsi Room</b>: Click on the `jitsi:` associated with the poster you would like to "present" or "attend". Again, each specific poster Jitsi room would open in a new "dedicated" tab or window. Clicking on it again would just lead you to that tab/window if it was already open. Presenters can share their poster window by clicking on the `Share your screen` button in the bottom-left corner of the Jitsi room. <!--Jitsi requires a browser extension to provide the ability to share your desktop. We have been using Jitsi happily on Chrome/Chromium for years. <a href="https://chrome.google.com/webstore/detail/jitsi-meetings/kglhbbefdnlheedjiejgomgmfplipfeb?hl=en-US">Extension link</a>.-->
+
+*Remark*: Unfortunately, the "source" spreadsheet neither had presentation dates/times, nor URLs to PDFs. So we might improve upon that (send a PR if you see how).
+
+### :heavy_check_mark: How to Do a Pull Request? Adding Your Poster's Info to the Table
+
+1. <b>GitHub account</b>: Create a <a href="https://github.com">GitHub account</a> if you don’t already have one. As a student, you can apply for the <a href="https://help.github.com/en/github/teaching-and-learning-with-github-education/applying-for-a-student-developer-pack">GitHub Student Developer Pack</a>, which includes offers and benefits from GitHub partners.
```

Comment:
Is there any specific reason why markdown annotation was replaced with pure HTML? like `<b>TEXT</b>` should be just `**TEXT**`

Reply:
Within the bullet points, there is no specific reason for the change to HTML. The HTML use is only needed in those parts where I've used `<p align=justify>` as I find it nicer to read. But that's a personal opinion. 😄 Anywhere else, I can change the HTML bits back to the Markdown annotation. 👍

Reply:
HTML --> Markdown (within the bullet points) done in [`4e402ea`](https://github.com/sofievdbos/ohbm2020-posters/commit/4e402ea81045f7b6ee5b3dd36cbb922652bec6e5) for: 1) text in bold & 2) URLs. Resizing images was not working when using the Markdown annotation, so I kept those in HTML style. I also cleaned some of my previous attempts (which were commented).